clients/unshaded/src/main/java/tachyon/client/UnderStorageType.java

```diff
@@ -22,10 +22,13 @@
  */
 public enum UnderStorageType {
   /** Persist data to Under Storage synchronously. */
-  PERSIST(1),
+  SYNC_PERSIST(1),
 
   /** Do not persist data to Under Storage. */
-  NO_PERSIST(2);
+  NO_PERSIST(2),
+
+  /** Persist data to Under Storage asynchronously via lineage */
+  ASYNC_PERSIST(3);
```

Comment:
In fact async persist can be used for other purposes, e.g. non-blocking UI. I'll update the comment.